Makefile

```diff
@@ -44,10 +44,6 @@ all: build
 dev:
 	$(DEV_ENV_CMD_IT) bash
 
-.PHONY: validate-generated
```

Comment:
I think this should stay in acs-engine until OpenShift support is removed.

Reply:
Again, because OpenShift isn't active, this build task is already obsolete.

Reply:
ok to remove